chef_master/source/knife_common_options.rst

```diff
@@ -1,5 +1,5 @@
 =====================================================
-Common Options
+Knife Common Command Line Options
```

Comment:
Agreed that this needs a change, however I prefer "Knife Common Options" as I don't think it's necessary to specify the command line bit.